frontend/src/reducers/expenses.js

```diff
@@ -3,6 +3,7 @@ import * as constants from '../constants/expenses';
 
 export default function expenses(state={}, action={}) {
   switch (action.type) {
+    case constants.PENDING_EXPENSES_SUCCESS:
```

Comment:
it would be more readable to add the verb at the beginning of those constants, e.g. `FETCH_PENDING_EXPENSES_SUCCESS`